sqlx-core/src/postgres/types/time.rs

```diff
@@ -0,0 +1,345 @@
+use std::convert::TryInto;
+use std::mem;
+
+use time::{date, offset, Date, NumericalDuration, OffsetDateTime, PrimitiveDateTime, Time};
+
+use crate::decode::{Decode, DecodeError};
+use crate::encode::Encode;
+use crate::postgres::protocol::TypeId;
+use crate::postgres::types::PgTypeInfo;
+use crate::postgres::Postgres;
+use crate::types::HasSqlType;
+
+const POSTGRES_EPOCH: PrimitiveDateTime = date!(2000 - 1 - 1).midnight();
```

Comment:
this is how rustfmt preferes this macro to look :(

Reply:
You can add `#![rustfmt::skip::macros(date)]` to the module/crate to make it not do this. Unfortunately it doesn't look like we can tune the actual style around binop spacing in Rustfmt to make it enforce the compact spacing for this macro: https://rust-lang.github.io/rustfmt/

Reply:
sadly it doesn't work "recursively" — attribute added to module has no effect in tests. Adding it in each test function works, but looks like bad solution :)

tbh, I think to leave it as is, if that's fine :)